assets/out

```diff
@@ -68,10 +90,14 @@ if [ ! "${encryption_key}" = "" ];then
 fi
 
 delete_flag=""
-if [ -z "${no_delete}" -o "${no_delete}" = "false" ]; then
+if [ -z "${no_delete}" ] || [ "${no_delete}" = "false" ]; then
   delete_flag="--delete"
 fi
 
+
+echo "Checking path in S3..."
+aws s3 ls "s3://${bucket}/${path}" --endpoint "${endpoint_url}" --no-verify-ssl > /dev/null || echo "Path check exited with code [$?], check path provided exists" && exit 1
```

Comment:
we have `endpoint_opt="--endpoint-url=${endpoint_url}"` earlier, it would be good to use it.
Also this is missing `${ssl_verification_opt}` but instead hardcoding it which is not correct.

Please check for this in all occurances of this command.